src/System.Reflection.Context/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      netfx;
```

Comment:
Why remove this? Actually how did it work in the first place if you used to have a placeholder _netfx configuration?